### ISSUE TITLE
After upgrade this morning: An unexpected error occured: System.BadImageFormatException: Could not load file or assembly 'CINTEROS.XRM.FETCHBUILDER.DLL'or bla bla

### ISSUE THREAD

**Issue by AlexanderRuiter** [MscrmTools/XrmToolBox#1098]
After upgrade XRMToolbox this morning the Tool does not start:
An unexpected error occured: System.BadImageFormatException: Could not load file or assembly 'CINTEROS.XRM.FETCHBUILDER.DLL'or bla bla

(BTW: occured should by occurred)

 XrmToolbox Version 
 - Tool version 1.2021.12.53

Windows 10

**Comment by AlexanderRuiter** [comment]
<img width="602" alt="image" src="https://user-images.githubusercontent.com/50696634/148176100-990d714c-6a30-4095-aa70-553f4d12fa97.png">

**Comment by AlexanderRuiter** [comment]
I unblocked all files in folder and subfolders but that did not help.

**Comment by MscrmTools** [comment]
This issue is already covered in the issue : https://github.com/MscrmTools/XrmToolBox/issues/1093
